firebase_auth_oauth_web/android/build.gradle

```diff
@@ -11,6 +11,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.google.gms:google-services:4.3.3'
```

Comment:
I am not sure if we need this here. We are not using the Android and iOS platforms so there's no benefit including this here.